modules/migration-mtc-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-6_{context}"]
+= {mtc-full} 1.8.6 release notes
+
+[id="technical-changes-1-8-6_{context}"]
+== Technical changes
+
+.Multiple migration plans for a namespace is not supported
+
+{mtc-short} version 1.8.6 and later do not support multiple migration plans for a single namespace.
+
+.VM storage migration
+
+VM storage migration feature changes from Technology Preview (TP) status to being Generally Available (GA).
+
+[id="resolved-issues-1-8-6_{context}"]
+== Resolved issues
+
+{mtc-short} 1.8.6 has the following major resolved issues:
+
+.UI crashes during a namespace search in the migration plan wizard
+
+When searching for a namespace in the *Select Namespace* step of the migration plan wizard, the user interface (UI) crashes and disappears after clicking *Search*. The browser console shows a JavaScript error indicating that an undefined value has been accessed. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1704[(MIG-1704)]
+
+.Unable to create a migration plan due to a reconciliation failure
+
+When creating a migration plan in {mtc-short}, the UI becomes stuck on the *Persistent Volumes* and you cannot continue. This issue occurs due to a critical reconciliation failure and returns a 404 API error when you attempt to fetch the migration plan from the backend. These issues cause the migration plan to remain in a *Not Ready* state, and you are prevented from continuing. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1705[(MIG-1705)]
+
+.Migration process becomes stuck after the `StageBackup` phase
+
+When migrating a Django and PostgreSQL application, the migration becomes stuck after the `StageBackup` phase. Even though all the pods in the source namespace are healthy before the migration begins, after the migration and when terminating the pods on the source cluster, the Django pod fails with a `CrashLoopBackOff` error. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1707[(MIG-1707)]
+
+.Migration shown as succeeded despite a failed phase due to a misleading UI status
+
+After running a migration using {mtc-short}, the UI incorrectly indicates that the migration was successful, with the status shown as *Migration succeeded*. However, the Direct Volume Migration (DVM) phase failed. This misleading status appears on both the *Migration* and the *Migration Details* pages. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1711[(MIG-1711)] 
+
+.Persistent Volumes page hangs indefinitely for namespaces without PVCs
```

Comment:
```suggestion
.Persistent Volumes page remains indefinitely for namespaces without PVCs
```
Same comment as above.

Reply:
The acronym PVC is not spelled out here, but is below. Not sure what the convention is here. Do you spell out on first instance for the topic, or for each bug?